### DISCUSSION TITLE
Need Help with Navidrome and Album Splitting Issue

### DISCUSSION THREAD

**Discussion by BobWs** [navidrome/navidrome#3323] [General]
Hello everyone,

I’m reaching out to see if anyone has encountered a peculiar issue with Navidrome and various client players. I’ve noticed that some of my music albums are being split into parts when accessed through different iOS players, while they appear as a single album in iTunes, which is how I expect them to be.

To give you some context, I use iTunes to organize and import my music, and my music folder is located on a Synology NAS. Additionally, I utilize a music tag editor on macOS to edit my music files before importing them into iTunes. I’ve checked the albums in the tag editor, and everything seems normal—nothing stands out that could be causing this issue.

Interestingly, when I use DS Audio, Synology's default music app, it recognizes the album as a single unit. This leads me to believe that the problem may be related to Navidrome and the client players, such as Play:Sub, Substreamer, and NavMus.

Has anyone else experienced something similar? If so, I would greatly appreciate any insights or solutions you might have to help me resolve this issue.
(See screenshots)

Thank you in advance for your assistance!

![IMG_0030](https://github.com/user-attachments/assets/27fa114b-3f87-46b5-b057-9d77a14cd276)
![IMG_0027](https://github.com/user-attachments/assets/b7baa8b3-812b-4090-a236-32c79f42aad5)

**Comment by deluan** [comment]
Albums are considered unique based on `AlbumArtist` and `Album` (name) tags. See https://www.navidrome.org/docs/faq/#-i-have-an-album-with-tracks-by-different-artists-why-is-it-broken-up-into-lots-of-separate-albums-each-with-their-own-artist

**Reply by BobWs** [reply]
`AlbumArtist` and `Album` are set unique. In iTunes the AlbumArtist and Album name are the same for all tracks in these Albums. Even In Tag Editor,  MusicBrainz Picard the tracks have the same info. I have also set `ND_SCANNER_GROUPALBUMRELEASES=true` and rebuild my docker container but nothing helps the Albums are still being split into parts. I have also trigger a full scan, didn't help.

Could this be a database problem? Maybe the database is holding on to older information and woudn't update the new info?

**Reply by deluan** [reply]
If changes in the files were detected, the DB should reflect the DB. Navidrome now also uses Release Date to disambiguate albums. Check if the dates are consistent as well.

**Reply by BobWs** [reply]
> If changes in the files were detected, the DB should reflect the DB. Navidrome now also uses Release Date to disambiguate albums. Check if the dates are consistent as well.

The release dates of the albums are all the same. In the screenshot example I also tried changing the date ("year") a few time to see if that would have effect, but unfortunately no soelaas still split up albums, and only in Navidrome.
Even when I sync music to an iphone the albums shows as one in Apple Music.

**Reply by deluan** [reply]
Can you send me one song from each "album", to help troubleshoot this? If you can, send it to `deluan @ navidrome . org`.

**Reply by BobWs** [reply]
Hi Deluan,
I've send you two emails as requested with one song from each "album" to help troubleshoot.
TIA

**Reply by deluan** [reply]
So, it took me a while to get time to look into this, sorry about that.

The tags are different. I tested two files:
```
./navidrome inspect music/\#3323\ -\ album\ splitting/Endless\ Love.mp3
Raw tags:
album = ['100 Hits - 80s Vol. 2']
'album artist' = ['Various Artists']
albumartist = ['Hits Collections']
date = ['1989']
...
Mapped tags:
AlbumArtistID = '03b645ef2100dfc42fa9785ea3102295'
AlbumArtist = 'Various Artists'
AlbumID = '50f24e013718f698cec5cec1ec23b899'
Compilation = false
```

```
./navidrome inspect music/\#3323\ -\ album\ splitting/Never\ Gonna\ Give\ You\ Up.mp3
Raw tags:
album = ['100 Hits - 80s Vol. 2']
albumartist = ['Hits Collections']
date = ['1989']
...
Mapped tags:
AlbumArtistID = 'e4c747e4a248ee00beda24a343ac74b9'
AlbumArtist = 'Hits Collections'
AlbumID = '44f22da816ad5f4f47ecdfd0adee627d'
Compilation = false

```

Seems that you have a "rogue" `album artist` tag, that is messing with Navidrome. You have two options:

1) Clear your tags. Remove all and add only the correct ones. I don't think you can remove _all_ tags with iTunes
2) Set `Compilation` tag to 1, which would also resolve the issue. 
![Screenshot 2024-10-18 at 7 36 26 PM](https://github.com/user-attachments/assets/08473df7-3e13-4b78-9426-9eb25e0fea00)

**Reply by BobWs** [reply]
Thanks @deluan for your time to investigate! I will try your suggestion. As you mentioned iTunes doesn’t remove all the tags, and in iTunes everything shows as one Album that’s why it difficult to resolve.

Do you know any other way or program I can use to correct my tags on a Mac?

**Reply by BobWs** [reply]
Yes it's solved! I used the trail version of this program https://www.mp3tag.de/en/index.html and within this app I saw the rogue `ALBUM ARTIST=VARIOUS ARTIST`. When I removed the tag and rescan Navidrome the Albums shows as one.

I don't understand how this could happen as I never saw this rogue tag in iTunes nor Tag Editor I used for years, guess my programs are too old!

Anyway thanks for al your help.

**Reply by deluan** [reply]
Awesome! By the way, you can use [MusicBrainz Picard](https://picard.musicbrainz.org/) for editing your tags, it is Open Source.
